MonoGame.Framework/Input/GamePad.UWP.cs

```diff
@@ -92,6 +92,14 @@ private static GamePadState PlatformGetState(int index, GamePadDeadZone deadZone
                 (state.Buttons.HasFlag(WGI.GamepadButtons.X) ? Buttons.X : 0) |
                 (state.Buttons.HasFlag(WGI.GamepadButtons.Y) ? Buttons.Y : 0) |
                 0;
+
+            // Check triggers
+            const double triggerThreshold = 0.1;
```

Comment:
I just came up with the 0.1 out of nowhere and it felt right when I tested. I'll update the value for the sake of parity with XInput.